python/cugraph/cugraph/tests/centrality/test_betweenness_centrality.py

```diff
@@ -522,24 +522,8 @@ def test_betweenness_centrality_nx(graph_file, directed, edgevals):
 
     Gnx = utils.generate_nx_graph_from_file(graph_file, directed, edgevals)
 
-    nx_bc = nx.betweenness_centrality(Gnx)
-    cu_bc = cugraph.betweenness_centrality(Gnx)
-
-    # Calculating mismatch
-    networkx_bc = sorted(nx_bc.items(), key=lambda x: x[0])
-    cugraph_bc = sorted(cu_bc.items(), key=lambda x: x[0])
-    err = 0
-    assert len(cugraph_bc) == len(networkx_bc)
-    for i in range(len(cugraph_bc)):
-        if (
-            abs(cugraph_bc[i][1] - networkx_bc[i][1]) > 0.01
-            and cugraph_bc[i][0] == networkx_bc[i][0]
-        ):
-            err = err + 1
-            print(f"{cugraph_bc[i][1]} and {networkx_bc[i][1]}")
-            print(f"{cugraph_bc[i][0]} and {networkx_bc[i][0]}")
-    print("Mismatches:", err)
-    assert err < (0.01 * len(cugraph_bc))
+    with pytest.raises(AttributeError):
+        cu_bc = cugraph.betweenness_centrality(Gnx)
```

Comment:
I haven't finished reviewing this PR but this did catch my eye so I'll comment now...
For this and all others with similar changes, we should just remove the entire test since it's no longer testing anything we support.

Reply:
That is the plan.   

first pass is to remove the code from the functions and keep the tests. 
Then cleanup the util files and remove nx functions
Then fix all the tests so that it checks for invalid input

Reply:
trying to not have one massive PR that touches everything